Cargo.toml

```diff
@@ -21,9 +21,14 @@ derive_serde_style = ["serde"]
 [dependencies]
 serde = { version="1.0.90", features=["derive"], optional=true }
 
-[target.'cfg(target_os="windows")'.dependencies.winapi]
-version = "0.3.4"
-features = ["consoleapi", "errhandlingapi", "fileapi", "handleapi", "processenv"]
+[dependencies.windows]
```

Comment:
Shouldn't that be `target.'cfg(target_os="windows")'.dependencies.windows`?

Reply:
Thankfully fixed by #18